component/src/main/java/com/celements/photo/unpack/UnpackComponent.java

```diff
@@ -87,6 +90,12 @@ public String unzipFileToAttachment(XWikiAttachment zipSrcFile, String attName,
         LOGGER.error("Exception while unpacking zip", ioe);
       } catch (XWikiException xwe) {
         LOGGER.error("Exception while unpacking zip", xwe);
+      } catch (DocumentSaveException dse) {
+        LOGGER.error("Exception saving unpacked file to destination document.", dse);
+      } catch (AttachmentToBigException atbe) {
+        LOGGER.error("Unpacked file to big.", atbe);
+      } catch (AddingAttachmentContentFailedException aacfe) {
+        LOGGER.error("Exception adding unpacked content to new attachment.", aacfe);
       } finally {
         if(newAttOutStream != null) {
```

Comment:
Please use Guava Closables.closeQuietly(newAttOutStream) instead to improve readability.